extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -428,4 +434,18 @@ private function getTrendingVideosData() {
 
 		return $data;
 	}
+
+	private function getWikiStats() {
+		global $wgCityId;
+
+		$wikiStats = [];
+		$service = new WikiDetailsService();
+		$wikiDetails = $service->getWikiDetails( $wgCityId );
+
+		if ( !empty( $wikiDetails['stats'] ) ) {
```

Comment:
I figured it is better to return empty array instead of undefined so we can avoid errors in: 
```
{{#if model.pages}}
```
because we are referencing key from object when object doesn't exists.